src/site/content/en/handbook/style/index.md

```diff
@@ -0,0 +1,158 @@
+  The primary guidelines for writing in the web.dev style.
+---
+
+The style guidelines apply to posts, codelabs, and anything else you might write on web.dev. They're intended to help both writers and reviewers evaluate drafts, and they're included in the [web.dev content checklist](/handbook/content-checklist).
+
+## Know your audience and purpose
+Have a specific [audience](/handbook/audience) in mind as you write.
+
+Make sure your content is meeting a need of your audience. (For example, is it teaching them how to do something? Helping them grasp a commonly misunderstood concept?)
+
+{% Compare 'worse', 'Don’t' %}
+> A post titled "The Media Capabilities API"
+
+If the reader isn't familiar with the API, its name isn't meaningful and doesn't convey how the post will be useful.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> A post titled "How YouTube improved video performance with the Media Capabilities API"
+
+While slightly longer, this title tells less experienced readers what they'll learn from the post while still providing the name of the API for readers who recognize it.
+{% endCompare %}
+
+## Be concise
+Use the fewest words possible while keeping language clear and [readable](/handbook/inclusion-and-accessibility#use-readable-language).
+
+{% Compare 'worse', 'Don’t' %}
+> Absolute and fixed positioning have been reimplemented and are more spec compliant than the old implementation. They also better match the behavior in other browsers.
+
+This sentence includes unnecessary words in some places (e.g., multiple phrases saying there's a new version), while not providing enough detail in others (e.g., which spec?).
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Absolute and fixed positioning are now more compliant with W3C specifications and better match the behavior in other browsers.
+
+This revision gives detail readers will likely need (e.g., the relevant specification) while omitting unnecessary words.
+{% endCompare %}
+
+Keep posts and codelabs short—typically 1,000 words or less.
+
+This emphasis on concision often means you can't discuss every aspect of a tool or technique. Instead, focus on the essential concepts required for readers to get started with a technique and apply it to their projects. If readers are interested in diving deeper, they can move on to resources outside of web.dev.
+
+## Chunk your content
+Dividing your content into logical chunks makes it easier to understand.
+
+Use section headings and subheadings to convey the structure of your piece. There are two structures that tend to work well on web.dev (though feel free to try others!):
+* **Measure → optimize → monitor**
+
+  This sequence works in many posts and is all but essential for posts about how to address Lighthouse audits.
+
+* **Concept → how-to → further reading**
+
+  This sequence works well for codelabs.
+
+Focus each paragraph (typically 2–3 sentences) on one main idea.
+
+Use a list for a set of instructions or related ideas.
```

Comment:
A requirement in many shops is that numbered lists are strictly reserved for instructions because it makes the instructions stand out from other content when scanning. Consider what happens when someone uses a phrase like, for example, 'there are three reasons' and follows it with numbered reasons. There's a likelihood that a subsequent editor will add reasons and forget to change the number in the intro. A better practice is to introduce with 'there are several reasons'  and follow it with bullets. I've seen this way too many times in 23 years.